src/components/elements/carousel.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+const Wrapper = styled.div`
+    padding: 5.4rem 0 5.8rem 8.7rem;
+    background-color: var(--color-white);
+    min-height: 41.8rem;
+    border-radius: 1rem;
+    position: relative;
+    height: 41.8rem;
+    box-sizing: border-box;
+    width: 100%;
+    max-width: 79.7rem;
+
+    .bullet-area {
+        position: absolute;
+        bottom: 5.8rem;
+
+        button {
+            margin-top: 0;
+            width: 1rem;
+            height: 1rem;
+            border-radius: 1rem;
+            margin-right: 1.25rem;
+            outline: none;
+            background-color: var(--color-grey-1);
+            border-color: var(--color-grey-1);
+        }
+        .active {
+            background-color: var(--color-black-4);
+            border-color: var(--color-black-4);
+        }
+    }
+    .background {
+        position: absolute;
+        right: 0;
+        bottom: 0;
+    }
+`
+
+class Carousel extends React.Component {
+    state = {
+        active_slide: this.props.slides[0],
+        active_bullet: 'bullet-0',
+    }
+    slidesClicked = e => {
+        this.setState({
+            active_slide: this.props.slides[e.target.name],
+            active_bullet: 'bullet-' + e.target.name,
+        })
+        e.target.add
+    }
+
+    render() {
+        const ActiveSlide = this.state.active_slide
+        const Background = this.props.background
+        const Bullet = props => (
+            <button
```

Comment:
For following the "styled components" way I think you could make this into a Button component and pass active as a prop and set style from that instead of using classNames. Same with the other classNames

Reply:
Done